frontend/src/action-creator.async.js

```diff
@@ -209,6 +209,21 @@ export const getUser = userId => async dispatch => {
   })
 }
 
+export const getUserPublicInformation = userId => async dispatch => {
+  return fetchWrapper({
+    url: `${FETCH_CONFIG.apiUrl}/users/${userId}/about`,
+    param: {
+      credentials: 'include',
+      headers: {
+        ...FETCH_CONFIG.headers
+      },
+      method: 'GET'
+    },
+    actionName: USER,
```

Comment:
this action name is already used by `getUser`, you must specify a more specific one

Reply:
Changed